docs/reference/ccr/requirements.asciidoc

```diff
@@ -0,0 +1,44 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-requirements]]
+=== Requirements for leader indices
+
+beta[]
+
+Cross-cluster replication works by replaying the history of individual write
+operations that were performed on the shards of the leader index. This means that the
+history of these operations needs to be retained on the leader shards so that
+they can be pulled by the follower shard tasks. The underlying mechanism used to
+retain these operations is _soft deletes_. A soft delete occurs whenever an
+existing document is deleted or updated. By retaining these soft deletes up to
+configurable limits, the history of operations can be retained on the leader
+shards and made available to the follower shard tasks as it replays the history
+of operations.
+
+Soft deletes must be enabled for indices that you want to use as leader
+indices. Enabling soft deletes requires the addition of some index settings at
+index creation time. You must add these settings to your create index
+requests or to the index templates that you use to manage the creation of new
+indices.
+
+IMPORTANT: This means that {ccr} can not be used on existing indices. If you have
+existing data that you want to replicate from another cluster, you must
+{ref}/docs-reindex.html[reindex] your data into a new index with soft deletes 
+enabled.
+
+[float]
+[[ccr-ov-soft-deletes]]
+==== Soft delete settings
+
+`index.soft_deletes.enabled`::
+
+Whether or not soft deletes are enabled on the index. Soft deletes can only be
+configured at index creation and only on indices created on or after 6.5.0.
```

Comment:
It's likely a good idea to state the default values for these settings. i.e. false in this case I think